frontend/apps/app/features/organizations/pages/OrganizationNewPage/OrganizationNewPage.tsx

```diff
@@ -0,0 +1,98 @@
+'use client'
+
+import { createClient } from '@/libs/db/client'
+import type { SupabaseClient } from '@/libs/db/server'
+import { urlgen } from '@/utils/routes'
+import { Button } from '@liam-hq/ui'
+import { useRouter } from 'next/navigation'
+import { type FC, type FormEvent, useState } from 'react'
+import styles from './OrganizationNewPage.module.css'
+
+export const OrganizationNewPage: FC = () => {
+  const [name, setName] = useState('')
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+  const router = useRouter()
+
+  const createOrg = async (supabase: SupabaseClient) => {
+    const { data, error } = await supabase
+      .from('Organization')
+      .insert({ name })
+      .select('id')
+      .single()
+
+    if (error) throw error
+    return data
+  }
+
+  const addUserToOrg = async (
+    supabase: SupabaseClient,
+    userId: string,
+    organizationId: number,
+  ) => {
+    const { error } = await supabase.from('OrganizationMember').insert({
+      userId,
+      organizationId,
+      status: 'ACTIVE',
+    })
+
+    if (error) throw error
+  }
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault()
+    if (!name) {
```

Comment:
nits:

As pr-agent suggests https://github.com/liam-hq/liam/pull/1296#issuecomment-2792327857 , organization `name`   value validation needed?

- e.g.
    - 3-20 characters
    - /[a-z][0-9]/i
         - allows hyphen? 
    - ...etc.

------

but this pr , ok!